src/sidebar/services/service-url.js

```diff
@@ -20,6 +20,7 @@ import * as urlUtil from '../util/url';
  * always returns empty strings as the URLs. After the links object has been
  * received from the API this function starts returning the real URLs.
  *
+ * @callback ServiceUrlGetter
  * @param {string} linkName - The name of the link to expand
  * @param {object} params - The params with which to expand the link
  * @returns {string} The expanded absolute URL, or an empty string if the
```

Comment:
Should we bother to clean up these in another pass? return vs returns?

Reply:
According to [the reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) either `@returns` or `@return` works, so we should just pick one of the two as a convention and stick to it.